src/sentry/options/defaults.py

```diff
@@ -157,6 +157,9 @@
 # regards to filter responses.
 register("store.lie-about-filter-status", default=False)
 
+# Skip nodestore save when saving an event
+register("store.save-event-skips-nodestore", default=True, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
Please ensure we keep this off when deploying and to ensure ops are aware when this is rolled out.

Reply:
Note here: before this goes live ops needs to register the option `store.save-event-skips-nodestore` to `False`.